EditView.vala

```diff
@@ -71,7 +71,12 @@ class EditView: Gtk.DrawingArea, Gtk.Scrollable {
 		blink_time = settings.get_int("cursor-blink-time") / 2;
 		can_focus = true;
 		set_has_window(true);
-		add_events(Gdk.EventMask.BUTTON_PRESS_MASK|Gdk.EventMask.BUTTON_RELEASE_MASK|Gdk.EventMask.BUTTON_MOTION_MASK|Gdk.EventMask.SCROLL_MASK|Gdk.EventMask.SMOOTH_SCROLL_MASK);
+		add_events(Gdk.EventMask.BUTTON_PRESS_MASK |
+			Gdk.EventMask.BUTTON_RELEASE_MASK |
+			Gdk.EventMask.BUTTON_MOTION_MASK |
+			Gdk.EventMask.SCROLL_MASK |
+			Gdk.EventMask.SMOOTH_SCROLL_MASK);
```

Comment:
It seems like having them on separate lines would be better for diffs when we might eventually change the widget's event set, but there's no reason to touch incidental formatting in this commit. Will do.